### ISSUE TITLE
Faster Whisper sometimes stops running suddenly

### ISSUE THREAD

**Issue by asusdisciple** [SYSTRAN/faster-whisper#620]
So I have large quantity of short 10-60 seconds audio files. I try to transcribe them with faster whisper in a loop. But sometimes whisper just stops/freezes all of a sudden. If I repeat the run it sometimes fails on the same file, sometimes it goes a bit further and fails at another file. I looked into the audio files and they seem to work just fine in any media player. I call faster whisper (large-v1) like this:
```
res = []
        for p in audiopath:  # filepaths
            segments, info = model.transcribe(p,
                                           beam_size=5
                                           task="transcribe",
                                           without_timestamps=True,
                                           temperature=[0], # temp 0 to make benchmark fair
                                           language=lang
                                           )
            logging.debug("Detected language '%s' with probability %f" % (info.language, info.language_probability))
            segments = list(segments)
            res.append("".join([i[4] for i in segments]))  #only append text to the result list
```

**Comment by Purfview** [comment]
Sometimes it stops without error when it's actually "out of memory".
How much VRAM there is in your GPU?

**Comment by sngazm** [comment]
I am experiencing the same problem. If you are using Windows, it may be related to this issue.
https://github.com/SYSTRAN/faster-whisper/issues/620

This issue does not provide a solution, but it does provide some workarounds.

**Comment by asusdisciple** [comment]
Not really, I use Ubuntu with an A100 and 80GB of RAM

**Comment by Fibs7000** [comment]
I experience the same issue with a T4 and 16GB of Memory. Suddenly stops and says start processing but them hangs.

It does not matter which files I use (have videos with different lengths) multiple container running faster whisper stop randomly.

**Comment by alexivaner** [comment]
me too facing the same problem

**Comment by llurk** [comment]
Has anyone encountered this issue and fixed it?

**Comment by Fibs7000** [comment]
Yes I have fixed it by checking every video via ffprobe and check if I get a duration. Basically we ignore all videos which would produce an error in ffmpeg and since then our service is super stable.

**Comment by llurk** [comment]
I found the simplest way to fix this issue is to clear the CMOS or reset the BIOS. I'm not entirely sure why this works, but recently I messed with the BIOS settings and my PC couldn't boot. After resetting the CMOS, the faster whisper error got fixed too.

**Comment by vizsatiz** [comment]
> Yes I have fixed it by checking every video via ffprobe and check if I get a duration. Basically we ignore all videos which would produce an error in ffmpeg and since then our service is super stable.

But does that mean something is wrong with the video, I am facing similar issue with larger audio files. But when I download and pay the audio it works fine in system audio players. Any solutions where I can still process these files
